README.md

```diff
@@ -171,43 +171,82 @@ Remember, every contribution counts. Let's make this project better together! 
 
 ## Set up development environment
 
-### Local setup
+### Developing dependencies
+
+- API
+  - Required packages:
+    - `golang` >= 1.22
+    - `g++`
+    - `libc-dev`
+    - `libheif` >= 1.15.1
+    - [go-face Requirements](https://github.com/Kagami/go-face#requirements) 
+        - `dlib`
+        - `libjpeg`
+        - `libblas` 
+        - `libcblas`, recommending using `libatlas-base`
+        - `liblapack`
+  - Optional tools during developing:
+    - `reflex`
+    - `sqlite`
+- UI
+  - Required packages:
+    - `node` = 18
+
+In Debian/Ubuntu, install dependencies:
 
-1. Install a local mysql server, and make a new database
-2. Rename `/api/example.env` to `.env` and update the `MYSQL_URL` field
-3. Rename `/ui/example.env` to `.env`
+```sh
+$ sudo apt install golang g++ libc-dev libheif-dev libdlib-dev libjpeg-dev libblas-dev libatlas-base-dev liblapack-dev # For API requirement
```

Comment:
always propose to run `sudo apt update` before installing something, as APT, in opposite to Brew, doesn't run the `update` if it wasn't run for long time

Reply:
I believe a Debian/Ubuntu user should know the basic usage of `apt`. It'll depend on their knowledge to run it.

Reply:
I'd like to show my follow-up PR to recommend users setting up development environment with Docker, see https://github.com/googollee/photoview/blob/dev-docker/README.md#set-up-docker-development-environment.

Only advanced user goes to local environment, and we could let them decide when to run `apt update`.

Reply:
> Only advanced user goes to a local environment

This might sense, but not necessary. Some users do not like the Docker dev workflow and prefer to build and test locally - this is just 1 example of when users will prefer to use local env even if they are not as skilled, as you assume, they should be)

Adding `apt update && ` prefix to installation commands wouldn't cost anything, but will solve a lot of potential issues

Reply:
Done.